cmd/cli/main.go

```diff
@@ -64,10 +64,10 @@ func main() {
 
 	// Root Flags
 	rootCmd.PersistentFlags().StringVarP(
-		&sdkConf.AuthURL,
+		&sdkConf.GroupsURL,
```

Comment:
Gropus is not a separate microservice, thus it should not have a configurable URL. It is either Auth (where it is today) or Things/Users (where it will be in the future).